### PR TITLE
update `pretty_name` for all datasets

### PR DESCRIPTION
**Pull request by bhavitvyamalik**
This PR updates `pretty_name` for all datasets. Previous PR #3498 had done this for only first 200 datasets

### CODE REVIEW

File: datasets/factckbr/README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - fact-checking
 paperswithcode_id: null
+pretty_name: factckbr
```
Comment (lhoestq) [inline datasets/factckbr/README.md:21]:
```suggestion
pretty_name: FACTCK BR
```

File: datasets/fquad/README.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ task_ids:
 - extractive-qa
 - closed-domain-qa
 paperswithcode_id: fquad
+pretty_name: French Question Answering Dataset
```
Comment (lhoestq) [inline datasets/fquad/README.md:26]:
```suggestion
pretty_name: "FQuAD: French Question Answering Dataset"
```

File: datasets/germaner/README.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - named-entity-recognition
 paperswithcode_id: null
+pretty_name: GermaNer
```
Comment (lhoestq) [inline datasets/germaner/README.md:21]:
```suggestion
pretty_name: GermaNER
```

File: datasets/glue/README.md [hunk 4]
```diff
@@ -64,6 +64,7 @@ task_ids:
   wnli:
   - text-classification-other-coreference-nli
 paperswithcode_id: glue
+pretty_name: General Language Understanding Evaluation benchmark
```
Comment (lhoestq) [inline datasets/glue/README.md:67]:
```suggestion
pretty_name: GLUE (General Language Understanding Evaluation benchmark)
```

File: datasets/harem/README.md [hunk 5]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - named-entity-recognition
 paperswithcode_id: null
+pretty_name: harem
```
Comment (lhoestq) [inline datasets/harem/README.md:21]:
```suggestion
pretty_name: HAREM
```

File: datasets/iwslt2017/README.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 paperswithcode_id: null
+pretty_name: IwslT217
```
Comment (lhoestq) [inline datasets/iwslt2017/README.md:3]:
```suggestion
pretty_name: IWSLT 2017
```

File: datasets/kde4/README.md [hunk 7]
```diff
@@ -109,6 +109,7 @@ task_categories:
 task_ids:
 - machine-translation
 paperswithcode_id: null
+pretty_name: kde4
```
Comment (lhoestq) [inline datasets/kde4/README.md:112]:
```suggestion
pretty_name: KDE4
```

File: datasets/kor_ner/README.md [hunk 8]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - named-entity-recognition
 paperswithcode_id: null
+pretty_name: KorNer
```
Comment (lhoestq) [inline datasets/kor_ner/README.md:21]:
```suggestion
pretty_name: KorNER
```

File: datasets/large_spanish_corpus/README.md [hunk 9]
```diff
@@ -80,6 +80,7 @@ task_categories:
 task_ids:
 - other-other-pretraining-language-models
 paperswithcode_id: null
+pretty_name: LargeSpanishCorpus
```
Comment (lhoestq) [inline datasets/large_spanish_corpus/README.md:83]:
```suggestion
pretty_name: The Large Spanish Corpus
```

File: datasets/lc_quad/README.md [hunk 10]
```diff
@@ -2,6 +2,7 @@
 languages:
 - en
 paperswithcode_id: lc-quad-2-0
+pretty_name: Largescale Complex Question Answering Dataset
```
Comment (lhoestq) [inline datasets/lc_quad/README.md:5]:
```suggestion
pretty_name: "LC-QuAD 2.0: Large-scale Complex Question Answering Dataset"
```

File: datasets/mwsc/README.md [hunk 11]
```diff
@@ -2,6 +2,7 @@
 languages:
 - en
 paperswithcode_id: null
+pretty_name: mwsc
```
Comment (lhoestq) [inline datasets/mwsc/README.md:5]:
```suggestion
pretty_name: The modified Winograd Schema Challenge (MWSC)
```

File: datasets/nchlt/README.md [hunk 12]
```diff
@@ -26,6 +26,7 @@ task_categories:
 task_ids:
 - named-entity-recognition
 paperswithcode_id: null
+pretty_name: nchlt
```
Comment (lhoestq) [inline datasets/nchlt/README.md:29]:
```suggestion
pretty_name: NCHLT
```

File: datasets/newspop/README.md [hunk 13]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - other-social-media-shares-prediction
 paperswithcode_id: null
+pretty_name: newspop
```
Comment (lhoestq) [inline datasets/newspop/README.md:21]:
```suggestion
pretty_name: News Popularity in Multiple Social Media Platforms
```

File: datasets/nkjp-ner/README.md [hunk 14]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - named-entity-recognition
 paperswithcode_id: null
+pretty_name: NkjpNer
```
Comment (lhoestq) [inline datasets/nkjp-ner/README.md:21]:
```suggestion
pretty_name: NJKP NER
```

File: datasets/piaf/README.md [hunk 15]
```diff
@@ -19,6 +19,7 @@ task_ids:
 - extractive-qa
 - open-domain-qa
 paperswithcode_id: null
+pretty_name: piaf
```
Comment (lhoestq) [inline datasets/piaf/README.md:22]:
```suggestion
pretty_name: Piaf
```

File: datasets/polsum/README.md [hunk 16]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - summarization
 paperswithcode_id: null
+pretty_name: polsum
```
Comment (lhoestq) [inline datasets/polsum/README.md:21]:
```suggestion
pretty_name: Polish Summaries Corpus
```

File: datasets/scielo/README.md [hunk 17]
```diff
@@ -27,6 +27,7 @@ task_categories:
 task_ids:
 - machine-translation
 paperswithcode_id: null
+pretty_name: scielo
```
Comment (lhoestq) [inline datasets/scielo/README.md:30]:
```suggestion
pretty_name: SciELO
```

File: datasets/swda/README.md [hunk 18]
```diff
@@ -18,6 +18,7 @@ task_categories:
 task_ids:
 - multi-label-classification
 paperswithcode_id: null
+pretty_name: swda
```
Comment (lhoestq) [inline datasets/swda/README.md:21]:
```suggestion
pretty_name: The Switchboard Dialog Act Corpus (SwDA)
```

File: datasets/thaisum/README.md [hunk 19]
```diff
@@ -20,6 +20,7 @@ task_ids:
 - language-modeling
 - summarization
 paperswithcode_id: null
+pretty_name: thaisum
```
Comment (lhoestq) [inline datasets/thaisum/README.md:23]:
```suggestion
pretty_name: ThaiSum
```

**Review by lhoestq — COMMENTED** [review]
Cool thank you :) 
I tried to improve some of them, the rest looks ok !

**Comment by bhavitvyamalik** [comment]
Pushed the lastest changes!

**Review by lhoestq — APPROVED** [review]
Thank you ! Merging now 🚀

**Pull request merged**